doc/14_data_frames.md

```diff
@@ -42,20 +42,18 @@ Example Changing Cluster/Keyspace Level Properties
 val conf = new SparkConf()
   .set("ClusterOne/spark.cassandra.input.split.size_in_mb","32")
```

Comment:
thinking more about this - I think I don't like this way of setting properties at all: `cluster:keyspace/propname`. I mean, these properties are very unusual and they cannot be set in spark-default.conf, because they do not follow `spark.*` pattern.
Since we want to fix multicluster, my idea is to put the cluster name and keyspace at the end of property name. This imho will be more natural:
- spark.cassandra.input.split.size_in_mb for everything
- spark.cassandra.input.split.size_in_mb.cluster for the whole cluster
- spark.cassandra.input.split.size_in_mb.cluster.keyspace for the whole keyspace

We could easily patch property name checker to such nuances and properties will be natural.
wdyt?